src/Modules/Manage/Dnn.PersonaBar.Users/Users.Web/src/_exportables/src/components/common/Password/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component,PropTypes  } from "react";
+import SingleLineInputWithError from "dnn-single-line-input-with-error";
+import Localization from "localization";
+import passwordStrength from "utils/PasswordStrength";
+import {CommonUsersService as UserService} from "../../../services";
+
+import "./style.less";
+
+class Password extends Component {
+    constructor(props) {
+        super(props);
+        this.passwordStrengthOptions = {
+            minLength:7,
+            minNumberOfSpecialChars:1,
+            validationExpression:""        
+        };
+    }
+
+    componentDidMount(){
+        
+        UserService.getPasswordStrengthOptions((data)=>{
+            this.passwordStrengthOptions = data;
+        });
```

Comment:
Is this common pattern in our components? I don't think that individual instances should be calling services this way